File: p242/p242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        from collections import Counter
        a = Counter(s)
        b = Counter(t)
        
        isFlag = True
        if len(a) == len(b):
            for k, v in a.items():
                if k not in b.keys() or v != b[k]:
                    isFlag = False
                    break
        else:
            isFlag = False
        return isFlag

File: p242/p242.cpp
#include "p242.hpp"


class Solution {
public:
    bool isAnagram(string s, string t) {
        return isAnagram_hash(s, t);
    }
};


int main(int argc, char* argv[]) {
    string s = "anagram";
    string t = "nagaram";
    Solution sol = Solution();
    cout << sol.isAnagram(s, t) << endl;
    return 0;
}